### ISSUE TITLE
Bundle doesn't include package config paths (*.json files) if the absolute path contains a '('

### ISSUE THREAD

**Issue by rsc1975** [systemjs/builder#811]
When I created the bundle in my linux env, the resulting js file doesn't include the *.json files (package config files). I opened a [Question](https://stackoverflow.com/questions/44453491/jspm-bundle-with-missing-json-files) in StackOverflow related with this issue, where I explain some details, but the problem is quite simple, my package config paths It's something like:

```
  packageConfigPaths: [
    "github:*/*.json",
    "npm:@angular/*.json",
    "npm:@covalent/*.json",
    "npm:*.json"
  ],
```

But my base directory It was something like : "`/home/tomcat/.jenkins/.../My Proyect (angular2)`", so after some debug, I see that the method `getPackageConfigPath(packageConfigPaths, normalized)` didn't return anything for the project libs (the same config worked ok in my local env), and I think that the problem is the regExp used to see if the packagename matches in the absolute path defined by `packageConfigPaths` section, the '(...)' in the absolute path is affecting the regExp `match` method.

I think that the regExp should be applied only over the `packageConfigPaths` values , but not on the entire absolute path, anyway I'm not sure, maybe there is a better solution.

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/builder/commit/2315f5faccf819c5153b2bcf223dc3a380527742 for the next release.

**Comment by guybedford** [comment]
Released in 0.16.7.
